vmngclient/session.py

```diff
@@ -202,6 +202,13 @@ def get_file(self, url: str, filename: Path) -> Response:
                 file.write(response.content)
         return response
 
+    def post_file(self, url: str, filename: Path, data: Dict = {}) -> Any:
+        headers = self.headers
+        del headers['Content-Type']
+        files = {'file': (filename.name, open(filename, 'rb'))}
+        response = self.post(url, data=data, files=files, headers={})
```

Comment:
request.headers items will overwrite session.headers items, no need to modify session.headers
In this case "content-type" needs to be specified in self.post(...headers=...)